### DISCUSSION TITLE
How can I revert change to signcolumn width from five characters

### DISCUSSION THREAD

**Discussion by aiman** [LazyVim/LazyVim#1633] [Q&A] [answered]
I don't override `signcolumn` (default `signcolumn=yes`) or folding  (default `nofoldenable`), but I do disable numbering (`nonumber` in effect).

Before upgrading from `7.0.0` to `9.5.0`, I count my signcolumn as just two characters wide, the number of characters left of the cursor. Now after upgrading to `9.5.0`, my signcolumn is five characters wide!

This is three characters too many I feel.

I can't find a way of shrinking it, I have tried with `signcolumn=auto:2` or `signcolumn=yes:2` but none of these settings seem to have any effect. The only thing that works is `signcolumn=no`, but I do want the signcolumn, just not as wide as five characters.

Any pointers appreciated.

(`NVIM v0.9.1`)

**Comment by folke** [comment]
~~`vim.opt.signcolumn = "auto"`~~

edit: `vim.opt.statuscolumn = ""`

**Reply by aiman** [reply]
I tried but still end up with a width of five chars

**Reply by folke** [reply, marked as answer]
Sorry, I meant `vim.opt.statuscolumn = ""`

**Reply by aiman** [reply]
At startup, with ~~`vim.opt.statuscolumn = ""`~~ `vim.opt.signcolumn = ""` set in `options.lua`, I see an error:

```
Failed loading config.options

vim/_meta.lua:0: E474: Invalid argument

# stacktrace:
...
```

I can attach a screenshot if that would be helpful.

But in any case, it seems like `signcolumn=yes` after this happens.

**Reply by folke** [reply]
`vim.opt.statuscolumn`  exists. Check the NEovim docs.
Only LazyVim change recently related to that is the custom `statuscolumn`.

Setting it to the empty string should disable that.

**Reply by aiman** [reply]
I misread the suggestion, sorry about that (have updated the comment)...

It seems that setting `vim.opt.statuscolumn = ""` works indeed, thank you!

**Reply by aiman** [reply]
For the record, I think its related to this change:

https://github.com/LazyVim/LazyVim/commit/364bcf325d91a06e6bd6516bdfed84399566cdb6
